content/en/continuous_integration/pipelines/custom.md

```diff
@@ -90,7 +91,19 @@ To send pipeline events programmatically to Datadog, ensure that your [`DD_API_K
    EOF
    {{< /code-block >}}
 
-1. After sending your pipeline event to Datadog, you can integrate additional event types such as `stage`, `job`, and `step`. For more information, see the [Send Pipeline Event endpoint][1].
+3. After sending your pipeline event to Datadog, you can integrate additional event types such as `stage`, `job`, and `step`. For more information, see the [Send Pipeline Event endpoint][1].
+
+## Running pipelines
+Pipelines sent with the `status` set to `running` should have the same `unique_id` as the final pipeline event. Additionally,
+running pipelines can be updated by adding more information while they are still running. The flow is as follows:
```

Comment:
```suggestion
Additionally, running pipelines can be updated by adding more information while they are still running. The flow is as follows:
```

Reply:
`The flow is as follows:` would seem to indicate that the following information is a series of steps. However, that doesn't seem to line up with the items, which don't direct a user to carry out any actions. Could this potentially be clarified?